src/CommunityToolkit.Aspire.Hosting.SqlDatabaseProjects/DacpacDeployer.cs

```diff
@@ -8,12 +8,12 @@ namespace CommunityToolkit.Aspire.Hosting.SqlDatabaseProjects;
 /// </summary>
 internal class DacpacDeployer : IDacpacDeployer
 {
-    /// <inheritdoc cref="IDacpacDeployer.Deploy(string, string, string, ILogger, CancellationToken)" />
-    public void Deploy(string dacpacPath, string targetConnectionString, string targetDatabaseName, ILogger deploymentLogger, CancellationToken cancellationToken)
+    /// <inheritdoc cref="IDacpacDeployer.Deploy(string, DacDeployOptions, string, string, ILogger, CancellationToken)" />
+    public void Deploy(string dacpacPath, DacDeployOptions options, string targetConnectionString, string targetDatabaseName, ILogger deploymentLogger, CancellationToken cancellationToken)
     {
-        var dacPackage = DacPackage.Load(dacpacPath, DacSchemaModelStorageType.Memory);
+        using var dacPackage = DacPackage.Load(dacpacPath, DacSchemaModelStorageType.Memory);
         var dacServices = new DacServices(targetConnectionString);
-        dacServices.Message += (sender, args) => deploymentLogger.LogInformation(args.Message.ToString());
-        dacServices.Deploy(dacPackage, targetDatabaseName, true, new DacDeployOptions(), cancellationToken);
+        dacServices.Message += (_, args) => deploymentLogger.LogDeploymentMessage(args.Message.ToString());
```

Comment:
What's the reason for this change? I'm not necessarily against it, but it feels unrelated?

Reply:
Sorry, just a little boy scout mentality. It is not related.
I got code analysis warnings locally on the unused parameter and on the logging call. So I refactored according to the latest practices with source generated logger messages for parformance. Reference: https://learn.microsoft.com/en-us/dotnet/core/extensions/high-performance-logging
But, in retrospect, the logging part is probably overkill given that this is never used in an running application, only for deployments. If you prefer, I can remove it to keep the code a little less complex and thus easier to maintain.

Reply:
I have added new commit undoing the unrelated logging changes  except for the adding "using" of the disposable `DacPackage`.